File: src/main/java/no/nav/syfo/controller/domain/NaermesteLeder.java
package no.nav.syfo.controller.domain;


import java.time.LocalDate;

public class NaermesteLeder {

    public String navn;
    public Long id;
    public String aktoerId;
    public String tlf;
    public String epost;
    public Boolean aktiv;
    public Boolean erOppgitt;
    public LocalDate fomDato;
    public String orgnummer;
    public String organisasjonsnavn;
    public LocalDate aktivTom;
    public Boolean arbeidsgiverForskuttererLoenn;

    public NaermesteLeder withAktoerId(final String aktoerId) {
        this.aktoerId = aktoerId;
        return this;
    }

    public NaermesteLeder withNavn(String navn) {
        this.navn = navn;
        return this;
    }

    public NaermesteLeder withTlf(String tlf) {
        this.tlf = tlf;
        return this;
    }

    public NaermesteLeder withEpost(String epost) {
        this.epost = epost;
        return this;
    }

    public NaermesteLeder withErOppgitt(Boolean erOppgitt) {
        this.erOppgitt = erOppgitt;
        return this;
    }

    public NaermesteLeder withAktiv(Boolean aktiv) {
        this.aktiv = aktiv;
        return this;
    }

    public NaermesteLeder withOrgnummer(final String orgnummer) {
        this.orgnummer = orgnummer;
        return this;
    }

    public NaermesteLeder withOrganisasjonsnavn(final String organisasjonsnavn) {
        this.organisasjonsnavn = organisasjonsnavn;
        return this;
    }

    public NaermesteLeder withId(Long id) {
        this.id = id;
        return this;
    }

    public NaermesteLeder withFomDato(LocalDate fomDato) {
        this.fomDato = fomDato;
        return this;
    }

    public Long hentId() {
        return id;
    }

    public NaermesteLeder withAktivTom(final LocalDate aktivTom) {
        this.aktivTom = aktivTom;
        return this;
    }

    public NaermesteLeder withArbeidsgiverForskuttererLoenn(final Boolean arbeidsgiverForskuttererLoenn) {
        this.arbeidsgiverForskuttererLoenn = arbeidsgiverForskuttererLoenn;
        return this;
    }
}

File: src/main/kotlin/no/nav/syfo/consumer/narmesteleder/NaermesteLeder.kt
package no.nav.syfo.consumer.narmesteleder

import java.io.Serializable
import java.time.LocalDate

class NaermesteLeder : Serializable {
    var navn: String? = null
    var id: Long? = null
    var aktoerId: String? = null
    var tlf: String? = null
    var epost: String? = null
    var aktiv: Boolean? = null
    var erOppgitt: Boolean? = null
    var fomDato: LocalDate? = null
    var orgnummer: String? = null
    var organisasjonsnavn: String? = null
    var aktivTom: LocalDate? = null
    var arbeidsgiverForskuttererLoenn: Boolean? = null
    fun withAktoerId(aktoerId: String?): NaermesteLeder {
        this.aktoerId = aktoerId
        return this
    }

    fun withNavn(navn: String?): NaermesteLeder {
        this.navn = navn
        return this
    }

    fun withTlf(tlf: String?): NaermesteLeder {
        this.tlf = tlf
        return this
    }

    fun withEpost(epost: String?): NaermesteLeder {
        this.epost = epost
        return this
    }

    fun withErOppgitt(erOppgitt: Boolean?): NaermesteLeder {
        this.erOppgitt = erOppgitt
        return this
    }

    fun withAktiv(aktiv: Boolean?): NaermesteLeder {
        this.aktiv = aktiv
        return this
    }

    fun withOrgnummer(orgnummer: String?): NaermesteLeder {
        this.orgnummer = orgnummer
        return this
    }

    fun withOrganisasjonsnavn(organisasjonsnavn: String?): NaermesteLeder {
        this.organisasjonsnavn = organisasjonsnavn
        return this
    }

    fun withId(id: Long?): NaermesteLeder {
        this.id = id
        return this
    }

    fun withFomDato(fomDato: LocalDate?): NaermesteLeder {
        this.fomDato = fomDato
        return this
    }

    fun hentId(): Long? {
        return id
    }

    fun withAktivTom(aktivTom: LocalDate?): NaermesteLeder {
        this.aktivTom = aktivTom
        return this
    }

    fun withArbeidsgiverForskuttererLoenn(arbeidsgiverForskuttererLoenn: Boolean?): NaermesteLeder {
        this.arbeidsgiverForskuttererLoenn = arbeidsgiverForskuttererLoenn
        return this
    }
}
